5_People/career.md

```diff
@@ -102,13 +102,21 @@ Everyone tracks working hours for themselves. At the end of each month, add your
 
 ## Severance Policy
 
+### Termination
+
 While many of us are good friends, we are still a business, not a family. Unlike a family, people join and leave us from time to time. Whenever we have to let someone go, this is how we do it.
 
-* The person needs to know at least 1 month in advance that their position could be canceled in the near term.
+* The person is given three notices, the third one being the final chance. There should be minimum one month in between notices, approved by one founder. A notice expires after six months.
+* The person needs to be fully aware that they are being issued a notice. These are documented on their [catchup note](https://github.com/teamniteo/handbook/blob/master/5_People/catchups.md).
+* On second notice, [productivity allowance](https://github.com/teamniteo/handbook/blob/master/5_People/benefits.md#productivity-allowance) is capped at €100 (any amount larger needs to be approved). The immediate supervisor should prepare and discuss with the Nitean the steps/tasks that they need to do to pick up their performance.
```

Comment:
```suggestion
* On the second notice, [productivity allowance](https://github.com/teamniteo/handbook/blob/master/5_People/benefits.md#productivity-allowance) is capped at €100. Any amount larger needs to be approved. The immediate supervisor should prepare and discuss with the Nitean the steps they need to take to fix the reason for the notice.
```
I also think we need to specify `capped at €100`.
Like, `capped at €100 per purchase` or `capped at €100 per month`.

Reply:
The regular allowance is €135, so I don't think we meant per month. Maybe total or per purchase (so you can't buy a computer for 2k).